src/main/io/vtx_smartaudio.c

```diff
@@ -710,7 +710,13 @@ bool vtxSmartAudioInit(void)
     const serialPortConfig_t *portConfig = findSerialPortConfig(FUNCTION_VTX_SMARTAUDIO);
     if (portConfig) {
         portOptions_e portOptions = SERIAL_STOPBITS_2 | SERIAL_BIDIR | SERIAL_BIDIR_PP_PD | SERIAL_BIDIR_NOPULL;
-
+#ifdef USE_SMARTAUDIO_NOPULLDOWN
```

Comment:
Should this be a run time config?

Reply:
This is a hack, probably only for nonstandard hardware. IMO it is desirable to make is slightly hard to use and have it in 'no support' category